docs/preview/02-Features/endpoint-validation.md

```diff
@@ -15,7 +15,7 @@ We provide support for endpoint validation, when implementing your own custom we
 The features described here require the following package:
 
 ```shell
-PM> Install-Package Arcus.EventGrid.WebApi.Security 
+PM> Install-Package Arcus.EventGrid.Security.WebApi
```

Comment:
Is this correct ?

Reply:
Yeah, I changed it so we can use the security namespace for web API and Azure Functions, as in:
* `.Security.WebApi`
* `.Security.AzureFunctions`
This is similar to other Arcus projects, as well, seeing how messaging, security, web API... repo's are structured.

Reply:
We haven't yet released this, so we are free to change it.